qemu/tests/timerdevice_boot.py

```diff
@@ -169,6 +182,15 @@ def update_clksrc(session, clksrc):
     if params["os_type"] == "linux":
         verify_timedrift(session, is_hardware=True)
 
+    repeat_nums = int(params["repeat_nums"])
+    if repeat_nums:
+        sleep_time = params["sleep_time"]
+        for index in range(repeat_nums):
+            time.sleep(int(sleep_time))
```

Comment:
For the first repeat,  sleep double `sleep_time`.  I think it can be moved after the first round finished.

Reply:
Line181-183 has run verify_timedrift, time.sleep here is ok.